__tests__/app.test.js

```diff
@@ -0,0 +1,49 @@
+const request = require("supertest"),
+  db = require("../db/connection.js"),
```

Comment:
This comma syntax is new to me and I just spent 10 minutes checking the docs to check it's not made up.
I like it! Though you should be consistent, using it for all consts (like the seed)

Reply:
lol, yeah i like it too, i hadnt thought to go back and edit the seed but i will